Scheduler/feature_scheduler/maintel/fbs_config_minimal_fieldsurvey.py

```diff
@@ -66,10 +67,10 @@ def get_scheduler():
         targets=targets, nside=nside, ntiers=1, band_to_filter=band_to_filter
```

Comment:
General comment, we're using the default nside now (https://github.com/lsst/rubin_scheduler/blob/9f72499f6f626bc2c35954af887fcf7303929358/rubin_scheduler/utils/constants.py#L5), so I think we should remove these overrides unless we expect to use something different. We should get in the habit of using default values now that we're using the full system.